src/main/java/exnihilo/ENItems.java

```diff
@@ -99,9 +107,17 @@ public static void registerItems() {
     GameRegistry.registerItem(HammerGold, ItemData.HAMMER_UNLOCALIZED_NAMES[3]);
     HammerDiamond = new ItemHammerDiamond();
     GameRegistry.registerItem(HammerDiamond, ItemData.HAMMER_UNLOCALIZED_NAMES[4]);
-    Crook = new ItemCrook();
-    GameRegistry.registerItem(Crook, "crook");
```

Comment:
will the removal of this cause issues with existing saves?

Reply:
Hmm yeah I didn't think about that it probably does. It gets re-named to `crook_wood`. There's no reason it couldn't be left as just `crook` though really.